### ISSUE TITLE
Plugin can't find config file

### ISSUE THREAD

**Issue by svetogam** [KoBeWi/Godot-Project-Builds#1]
I'm using the Linux version and Project Builder works when I run it. But when trying to run things through the plugin, I get the error message:

> Project Builder config file not found. Make sure you run Project Builder directly at least once.

Following the message's instructions didn't fix this.

Investigating the issue a little, I think I found the problem here:
```
func run_project_builder(routine := ""):
	var project_builds_config := FileAccess.open(EditorInterface.get_editor_paths().get_config_dir().path_join("app_userdata/Godot Project Builder/project_builds_config.txt"), FileAccess.READ)
```
Here `EditorInterface.get_editor_paths().get_config_dir()` returns `/home/<user>/.config/godot` for me. But my `app_userdata` folder is in `/home/<user>/.local/share/godot/` instead. I can find the file it's looking for at `/home/<user>/.local/share/godot/app_userdata/Godot Project Builder/project_builds_config.txt`. That is, it's in where I get to from Editor > Open Editor Data Folder instead of Editor > Open Editor Settings Folder.

**Comment by KoBeWi** [comment]
Should be fixed in 39fa063
See if it works for you.

**Comment by svetogam** [comment]
The short answer is it works now. The long answer is this log I kept. I'll leave it here since it might be useful for tracking down other bugs and releasing the next version without problems. It's not clear to me how much of this is relevant so I'm posting it all.

```
1. Copy-paste the commit diff to replace the line.
  - Project builder runs from the plugin. But now most tasks show <null> (the exception being sub-routines.)
  - When adding a task, most tasks show up as null. These are 1, 2, 4, 5, and 6 in the order of the menu.
  - Entering a routine and pressing back crashes the program.
2. Press Refresh Routine List.
  - Same problem.
  - Executing gives errors. "Some tasks are invalid!"
  - "Execute Routine" menu options from inside Godot give the same error as executing from the program.
3. Restart plugin
  - Same problem.
4. Restart Godot
  - Same problem.
5. Open the program outside the plugin. Then run from the plugin.
  - Tasks are fine outside like before and no problems found.
  - Problem persists when running from plugin.
* 6. Manually set "project_builder_path" in config file (which was empty "" before) and restart Godot
  - Problem is fixed.
* 7. (Back to non-working config file.) Rebuild Project Builder from source, run it, then run it from Godot.
  - Problem is fixed.
8. Delete config file. Run old version.
  - <null> problem is back.
* 9. Delete config file. Run new version.
  - Works.
10. Delete config file. Delete addon and replace with folder from source. Run new version.
  - Global "godot_path" goes to project builder instead of Godot.
  - Execution hangs.
* 11. Manually set global Godot path from inside Project Builder.
  - Works.
(Probably unrelated after 11.)
  - Rebuilding everything hangs on the step of exporting linux32. Though that step seems to have completed, it just doesn't move on. (I waited for about a minute.)
  - This happened once. But the next 2 times it did not happen.
```

**Comment by KoBeWi** [comment]
There should be some errors in the console. I also encountered the null bug, but it fixed itself and I had no way to investigate properly.

**Comment by svetogam** [comment]
I only got the null bug when I modified the plugin from the addons folder of my own project, so I couldn't see any errors in the console.

My guess is the problem derives from "project_builder_path" in the config file not being set. Since setting it manually solved the problem for me at the time. So I guess it's looking for the Tasks folder which it sometimes finds and sometimes doesn't if it's not set, but always finds when it is set.

But IDK. I don't really want to try to figure it out.
